modules/payment/src/Plugin/Commerce/CheckoutPane/PaymentInformation.php

```diff
@@ -51,6 +64,8 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
     parent::__construct($configuration, $plugin_id, $plugin_definition, $checkout_flow);
 
     $this->entityTypeManager = $entity_type_manager;
+    $this->paymentGatewayStorage = $this->entityTypeManager->getStorage('commerce_payment_gateway');
```

Comment:
You don't need the storages under $this when you already have the entityTypeManager.